### DISCUSSION TITLE
Deactivate showing stats with adding a + at the end of the shorturl

### DISCUSSION THREAD

**Discussion by oingoboingo** [YOURLS/YOURLS#3787] [Q&A] [answered]
Hi, 

haven't found this topic here. Is there a way to suppress displaying stats when a user is adding a + at the end of the shorturl?

Best,
Marc

**Comment by dgw** [comment, marked as answer]
If your instance is public, you can make stats private [with `YOURLS_PRIVATE_INFOS`](https://yourls.org/docs/guide/essentials/configuration#yourls_private_infos).

Any logged-in user can access the stats in either mode.

**Reply by oingoboingo** [reply]
Hi dgw, thanks. Damn it, I should've checked that first in an inkognito browser. Because I still was logged in. ;-)
